Makefile

```diff
@@ -1,12 +1,12 @@
-export OS_NAME       := $(shell uname)
```

Comment:
I don't find it to be an "improvement" to have one `export` declaration for all variables instead of one per variable.

Reply:
The only variable which doesn't appear to need the export is `$(OS_NAME)`; both `$(OS_ARCH)` and `$(NO_SUDO)` may be used by child processes.

Revert the change; it won't matter if we export `$(OS_NAME)`.